spec/v1.0-draft/digest_set.md

```diff
@@ -0,0 +1,63 @@
+# DigestSet field type specification
+
+Version v1.0-draft
+
+Set of one or more cryptographic digests for a single software artifact or
+metadata object.
+
+## Schema
+
+```json
+{
+  "<ALGORITHM_1>": "<HEX/BASE64 VALUE>",
+  "<ALGORITHM_2>": "<HEX/BASE64 VALUE>",
+  ... 
+}
+```
+
+## Fields
+
+A DigestSet is represented as a _JSON object_ mapping algorithm name to
+lowercase hex-encoded value. Usually there is just a single key/value pair,
+but multiple entries MAY be used for algorithm agility.
+
+> **TODO**: Add language for supporting base64-encoded digest values (re: goModuleH1).
+
+Supported algorithms:
+
+-   Standard cryptographic hash algorithms, for cases when the method
+    of serialization is obvious or well known:
+    `sha256`, `sha224`, `sha384`, `sha512`, `sha512_224`, `sha512_256`,
```

Comment:
I prefer disambiguating between SHA-2 and SHA-3, but I guess, at this point, the differences are clear to the initiated at least. OTOH, it might break existing verifiers.

Reply:
This uses the NIST names: https://csrc.nist.gov/projects/hash-functions (converting to lowercase and replacing `-` with `_`)

Reply:
If there are any outstanding points to be resolved, I think we can do this as part of the broader DigestSet terminology clarifications we have lined up (per #130).